test/mocha/test.js

```diff
@@ -0,0 +1,6 @@
+var assert = require('assert');
+describe('Mocha Test', function () {
+  it('dummy test', function () {
+    assert.equal(true, true);
```

Comment:
you might use assertTrue(true), but it doesn't really matter, its more like  a comment than a change request.

Reply:
checked node.js doc/
no assertTrue
but assert.ok does same work